monitoring/ceph-mixin/README.md

```diff
@@ -31,13 +37,21 @@ alert rules that should provide a decent set of default alerts for a
 Ceph cluster. Just put this file in a place according to your Prometheus
 configuration (wherever the `rules` configuration stanza points).
 
+
+##### Recommended versions: 
+-prometheus v2.7.2
+
 #### SNMP
 Ceph provides a MIB (CEPH-PROMETHEUS-ALERT-MIB.txt) to support sending Prometheus
 alerts through to an SNMP management platform. The translation from Prometheus
```

Comment:
While we're here, I think "through" is superfluous.

Reply:
I think you're right. 

I've removed it.